File: Entrega-1/src/entregaLabo/ConcreteStrategyInCircles.java
package entregaLabo;

import robocode.JuniorRobot;

public class ConcreteStrategyInCircles extends Strategy{
	
	
	public ConcreteStrategyInCircles(JuniorRobot robot) {
		super(robot);
	}

	@Override
	public void move() {
		robot.ahead(600);
		robot.bearGunTo(90);
	}

	@Override
	public void avoid() {
		robot.turnRight(10);
		robot.ahead(10);
	}
	
}

File: Entrega-3/ConcreteStrategyInCircles.kt
package entregaLabo

import robocode.JuniorRobot

class ConcreteStrategyInCircles(robot: JuniorRobot) : Strategy(robot) {

    override fun move() {
        robot.ahead(600.0)
        robot.bearGunTo(90)
    }

    override fun avoid() {
        robot.turnRight(10.0)
        robot.ahead(10.0)
    }
}
